src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-TRAINING.json

```diff
@@ -0,0 +1,95 @@
+{
+    "Description": "MLPerf Training GPU Performance Workload",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "10:00:00",
+        "SupportedPlatforms": "linux-x64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "SpecialRequirements": "This is an Nvidia GPU-specialized workload. It depends upon the system having an Nvidia GPU card/chip. Requires an OSDisk of >=256GB size and Data disk of >=8TB size"
+    },
+    "Parameters": {
+        "Username": "",
+        "DiskFilter": "SizeGreaterThan:8000gb",
+        "LinuxCudaVersion": "12.0",
+        "LinuxDriverVersion": "525",
+        "LinuxLocalRunFile": "https://developer.download.nvidia.com/compute/cuda/12.0.0/local_installers/cuda_12.0.0_525.60.13_linux.run"
+    },
+    "Actions": [
+        {
+            "Type": "MLPerfTrainingExecutor",
+            "Parameters": {
+                "Scenario": "training-mlperf-bert-batchsize-45-gpu-8",
```

Comment:
By gpu 8, do we mean this scenario is meant for 8 GPU vm sku/SUT? It means that we can change number of gpus and batchsize and it will work on further vm skus also, correct? Or we are not aware of it yet?

Reply:
Correct 
We can control the number of GPUs utilized.
Also for 1 GPU size VM we have separate configuration name.